tests/components/netatmo/test_webhook.py

```diff
@@ -1,37 +1,10 @@
 """The tests for Netatmo webhook events."""
-from homeassistant.components.netatmo.const import DATA_DEVICE_IDS, DATA_PERSONS
 from homeassistant.components.netatmo.webhook import async_handle_webhook
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.util.aiohttp import MockRequest
 
 
-async def test_webhook(hass):
-    """Test that webhook events are processed."""
-    webhook_called = False
-
-    async def handle_event(_):
-        nonlocal webhook_called
-        webhook_called = True
-
-    response = (
-        b'{"user_id": "123", "user": {"id": "123", "email": "foo@bar.com"},'
-        b'"push_type": "webhook_activation"}'
-    )
-    request = MockRequest(content=response, mock_source="test")
-
-    async_dispatcher_connect(
-        hass,
-        "signal-netatmo-webhook-None",
-        handle_event,
-    )
-
-    await async_handle_webhook(hass, "webhook_id", request)
-    await hass.async_block_till_done()
-
-    assert webhook_called
-
-
-async def test_webhook_error_in_data(hass):
+async def test_webhook_error_in_data(hass, config_entry):
```

Comment:
Can we remove this test?

Reply:
Yes, this was only added to raise coverage. I'll remove it.